docs/snippets/compile_model.cpp

```diff
@@ -0,0 +1,10 @@
+
+{
+//! [compile_model]
+
+#include <openvino/runtime/core.hpp>
+
+ov::Core.compile_model(device, modelPath, properties).export_model(compiled_blob);
+
+//!  [compile_model]
+}
```

Comment:
```suggestion
//! [compile_model]

#include <openvino/runtime/core.hpp>

ov::Core.compile_model(device, modelPath, properties).export_model(compiled_blob);

//!  [compile_model]
```
Also propose to rename snippet name to `export_compiled_mode`